test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpExporterOptionsTests.cs

```diff
@@ -24,10 +67,10 @@ public void OtlpExporterOptions_Defaults()
     {
         var options = new OtlpExporterOptions();
 
-        Assert.Equal(new Uri("http://localhost:4317"), options.Endpoint);
+        Assert.Equal(new Uri(OtlpExporterOptions.DefaultGrpcEndpoint), options.Endpoint);
```

Comment:
I think it's better to use the hardcoded string `"http://localhost:4317"` for the test instead of relying on the constant defined in `src` file otherwise the test would never complain even if the constant defined in the `src` file accidentally gets changed.

Reply:
We've debated this before 🤣 I subscribe to the DRY/single source of truth principle. If a dev wants to change the value, it should be defined in a single spot. If a dev changes the constant sure the tests might break. Or that dev could just update the tests too. Either case it is going to have to pass code review. Have we encountered issues with people trying to change our constants to be incorrect?

Reply:
> Either case it is going to have to pass code review. Have we encountered issues with people trying to change our constants to be incorrect?

I feel, when possible, having unit tests that signal someone that they messed up is better. It's much quicker than the issue being caught in a PR review. It also slightly reduces the burden on the reviewer.


> We've debated this before 🤣

Haha yes. I remember now. I'll leave this up to you.

Reply:
> > Either case it is going to have to pass code review. Have we encountered issues with people trying to change our constants to be incorrect?
> 
> I feel, when possible, having unit tests that signal someone that they messed up is better. It's much quicker than the issue being caught in a PR review. It also slightly reduces the burden on the reviewer.

+1, I don't have super strong opinion here, but my preference is: if there is a behavior change in the production code, it should be caught by unit test. In other words, if a dev wants to make a behavior change, both production code and unit test code need to be updated.